### ISSUE TITLE
Google Chrome seems to think there is a visible Menu Bar when reconnecting an external display.

### ISSUE THREAD

**Issue by lgomez** [eczarny/spectacle#645]
Hello,

Just wanted to report this here as I don't think anyone has already.

Notes:
- Seems to apply to Google Chrome only but there may be other applications with the same issue. iTerm, Atom, Sketch 3 and, at least, Safari, do not show this.
- This is only visible on the external display.
- The issue presents itself when connecting the laptop to the external display.

Summary

Whenever I reconnect the laptop to my external display, Google Chrome stays as it was before connecting the display but as soon as I come back to it (return focus to it), the top border drops to make space for the Menu Bar even though the Menu Bar is hidden. Additionally, if I try to resize Google Chrome using Spectacle, it will respond by resizing to half the screen width, still not going all the way to the top but leaving the top-menu space and stop responding to subsequent Spectacle commands.

In short, Google Chrome's behavior is broken when reconnecting an external monitor.

Thanks,

**Comment by eczarny** [comment]
Chrome has historically been a problem for Spectacle. Updates to the browser have been known to break OS X behavior (this issue sounds a bit familiar as well). Unfortunately there really isn't much to do here. Working around the issue doesn't make much sense since that would then break once the underlying issue in Chrome is addressed.
